src/components/access-type.js

```diff
@@ -551,6 +557,8 @@ const mapDispatchToProps = dispatch => ({
  *                  email={email}
  *                  phone={phone}
  *                  disableMetering={disableMetering}
+ *                  prodHost="www.abc.com"
+ *                  stagingHost="staging.abc.com"
```

Comment:
You need to give a short explanation of what's `prodHost` and `stagingHost`.. like how it's given on line 514 to 519

Reply:
@nandap4790 Default value as in?

Reply:
what is the fallback value if there is no value passed from the frontend

Reply:
@cricket-lover 
```
 *  propName    |   Description
 *  --------    |   -----------
 *  prodHost    |   It is passed to configure the production host.(Optional, default value is "https://www.accesstype.com")
 *  stagingHost |   It is passed to configure the staging host.(Optional, default value is "https://staging.accesstype.com")
 *
```